sdk/ai/azure-ai-projects/azure/ai/projects/models/_enums.py

```diff
@@ -304,12 +304,7 @@ class MessageStreamEvent(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 
 
 class OpenApiAuthType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
-    """
-    Authentication type for OpenApi endpoint. Allowed types are:
-    - Anonymous (no authentication required)
-    - Connection (requires connection_id to endpoint, as setup in AI Foundry)
-    - Managed_Identity (requires audience for identity based auth)
-    """
+    """Type of chunking strategy."""
```

Comment:
"""Type of chunking strategy.""" comments seems not correct

Reply:
Fixed.